cmd/restic/cmd_backup.go

```diff
@@ -543,7 +543,11 @@ func runBackup(ctx context.Context, opts BackupOptions, gopts GlobalOptions, ter
 	if !gopts.JSON {
 		progressPrinter.V("load index files")
 	}
-	err = repo.LoadIndex(ctx)
+
+	bar := newTerminalProgressMax(!gopts.Quiet, 0, "index files loaded", term)
+	defer bar.Done()
```

Comment:
`bar.Done()` must be called once `repo.LoadIndex` has completed, not at the end of the `runBackup` function.